e2etest.ps1

```diff
@@ -0,0 +1,61 @@
+# Define the functions
+Function RemoveFileIfExist($fileName) {
+    if (Test-Path $fileName) {
+        Remove-Item -Force $fileName
+    }
+}
+
+Function RemoveFolderIfExist($folderName) {
+    if (Test-Path $folderName) {
+        Remove-Item -Recurse -Force $folderName
+    }
+}
+
+Function DownloadFileFromUrl($url, $destination) {
+    $wc = New-Object System.Net.WebClient
+    $wc.DownloadFile($Env:FUNCTIONCLI_URL, $functionCLIZipPath)
+}
+
+Function UpdateMavenPluginVersion($pomLocation, $version) {
+    $PomFile = gi $pomLocation
+    $Pom = [xml](gc $PomFile)
+    $Pom.project.properties.'azure.functions.maven.plugin.version' = $functionVersion
+    $Pom.Save($PomFile.Fullname)
+}
+
+#Scripts
+$base = pwd
+$functionCLIPath = "$base\Azure.Functions.Cli"
+$functionCLIZipPath = "$base\Azure.Functions.Cli.zip"
```

Comment:
`$functionCLIZipPath = $functionCLIPath +".zip"`. Once the function cli path changes, there is only one place to change.